integration/tinycam/custom_vendors.xml

```diff
@@ -1,15 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <vendors>
-    <vendor name="Xiaomi">
```

Comment:
Really, there cannot be different models from one vendor?

Reply:
unfortunately not. Tinycam complains and doesn't read the file.